_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -922,27 +1051,92 @@ accounts (e.g. dev, stage, and production) where your applications run.
-* link:https://github.com/gruntwork-io/module-asg[`module-asg`] applies IAM roles to instances in auto-scaling
+* link:https://github.com/gruntwork-io/terraform-aws-asg[`terraform-aws-asg`] applies IAM roles to instances in auto-scaling
  group.
 * link:https://github.com/gruntwork-io/terraform-aws-eks/blob/master/modules/eks-cluster-workers/main.tf[`terraform-aws-eks`]
  uses IAM roles for EKS cluster workers.
-* link:https://github.com/gruntwork-io/module-ecs/tree/master/modules/ecs-cluster[`ecs-cluster`] creates IAM
+* link:https://github.com/gruntwork-io/terraform-aws-ecs/tree/master/modules/ecs-cluster[`ecs-cluster`] creates IAM
  roles for ECS instances
-* link:https://github.com/gruntwork-io/package-lambda/tree/master/modules/lambda[`lambda`] creates IAM
+* link:https://github.com/gruntwork-io/terraform-aws-lambda/tree/master/modules/lambda[`lambda`] creates IAM
  roles for Lambda functions
 
 Use these modules whenever possible. You should always use IAM roles in your own modules any time you need to provide access to the AWS API. Using static API credentials should be avoided whenever possible.
 
+[[cleanup_expired_certs_deployment]]
+=== Automatically remove expired SSL/TLS certificates from IAM
+Next, use the link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/modules/cleanup-expired-certs/README.adoc[`cleanup-expired-certs`]
+module to deploy a Lambda function to remove all expired SSL/TLS certificates from AWS IAM. This module will add all of the relevant IAM resources needed to
+periodically execute the Lambda function and optionally, report metrics to CloudWatch. Be sure to deploy this module into each of your AWS accounts.
+
+If your using Terragrunt, then you can invoke the standalone module directly by following the pattern outlined in
+<<wrapper_code_terragrunt>> and creating a `terragrunt.hcl` file with the following contents:
+
+.infrastructure-live/<your account>/_global/cleanup-expired-certs/terragrunt.hcl
+[source,hcl]
+----
+terraform {
+  # Make sure to replace <VERSION> in this URL with the latest terraform-aws-cis-service-catalog release
+  source = "git::ssh://git@github.com/gruntwork-io/terraform-aws-cis-service-catalog.git//modules/cleanup-expired-certs?ref=<VERSION>"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+inputs = {
+  schedule_expression                = "rate(1 hour)"
+  report_cloudwatch_metric           = true
+  report_cloudwatch_metric_namespace = "custom/cis"
+  report_cloudwatch_metric_name      = "cleanup-expired-iam-certs-count"
+}
+----
+
+[[iam_access_analyzer_deployment]]
+=== Steps to enable IAM Access Analyzer for all enabled AWS regions
+To enable IAM Access Analyzer in your AWS Account or Organization, you need to do it separately for every region.
+
+To enable this for your AWS account:
+* use the examples provided as part of `learning-and-testing/landingzone`:
```

Comment:
NIT: This `*` is not rendering as a bullet point: https://deploy-preview-392--keen-clarke-470db9.netlify.app/guides/compliance/how-to-achieve-cis-benchmark-compliance/#iam_access_analyzer_deployment

Reply:
Actually, this whole sentence should be removed, as CIS customers should NOT be using the `acccount-baseline` modules, as those don't take into account CIS requirements. Instead, they should solely use the `iam-access-analyzer` module as shown below.

Reply:
ahaa! noted!

Reply:
removed :) good shout!